lti_consumer/lti_xblock.py

```diff
@@ -1465,7 +1494,10 @@ def _get_context_for_template(self):
         allowed_attributes = dict(bleach.sanitizer.ALLOWED_ATTRIBUTES, **{'img': ['src', 'alt']})
         sanitized_comment = bleach.clean(self.score_comment, tags=allowed_tags, attributes=allowed_attributes)
 
-        if self.lti_version == 'lti_1p1':
+        if self.config_type == "external":
+            self.launch_url = self._get_lti_consumer().lti_launch_url
+
+        if self.lti_version == 'lti_1p1' or self.config_type == 'external':
```

Comment:
Can you add a comment here explaining why we need to check if it's external for now?